newChecklist.md

```diff
@@ -1,155 +0,0 @@
-1. [CODE STYLE] - Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search)
-
-BAD EXAMPLE:
-```jsx
```

Comment:
Consider using more descriptive variable names in code examples. For example, instead of `setVariable`, consider `setUserName`. 🤔